### PR TITLE
Filling layout and routes

### PR DESCRIPTION
**Pull request by vietnh3005**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -4,4 +4,3 @@ def full_title page_title = ""
     page_title.empty? ? base_title : "#{page_title} | #{base_title}"
     end
   end
```
Comment (DuongNT1512) [inline app/helpers/application_helper.rb:5]: indent

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+    <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    sample application.
+  </h2>
+
+  <%= link_to "Sign up now!", signup_path, class: "btn btn-lg btn-primary" %>
+</div>
+
+<%= link_to image_tag("rails.png", alt: "Rails logo"),
+  "http://rubyonrails.org/" %>
+
```
Comment (DuongNT1512) [inline app/views/static_pages/home.html.erb]: xoas dòng

File: config/routes.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   root "static_pages#home"
-  get  "static_pages/home"
-  get  "static_pages/help"
-  get  "static_pages/about"
-  get  "static_pages/contact"
+  get  "/help", to: "static_pages#help"
```
Comment (DuongNT1512) [inline config/routes.rb]: indent

File: app/views/layouts/_rails_default.html.erb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+    <%= csrf_meta_tags %>
```
Comment (hellovietnam93) [inline app/views/layouts/_rails_default.html.erb]: ident

**Pull request merged**